plover_engine_server/websocket/server.py

```diff
@@ -18,7 +19,8 @@
 class WebSocketServer(EngineServer):
     """A server based on WebSockets."""
 
-    def __init__(self, host: str, port: str):
+    _ssl: dict
+    def __init__(self, host: str, port: str, ssl: dict):
```

Comment:
`dict[str, str]` might be better (although depends on Python version supposed `from typing import Dict` may be needed)

Reply:
Yes, sure. I have changed to:

```python
    _ssl: dict[
        { "cert_path" , str },
        { "key_path", str }
    ]
```

Reply:
Actually is this valid Python type annotation syntax? (The proper way is `TypedDict` I think?) https://docs.python.org/3/library/typing.html#typing.TypedDict

Reply:
> Actually is this valid Python type annotation syntax? (The proper way is `TypedDict` I think?) https://docs.python.org/3/library/typing.html#typing.TypedDict

I researched on that and again you are right. TypeDict is the correct approach.